text/0000-unsafe-fields.md

```diff
@@ -0,0 +1,137 @@
+- Feature Name: `unsafe_fields`
```

Comment:
The appears to be missing the https://github.com/rust-lang/rfcs/blob/master/0000-template.md#rationale-and-alternatives section.  Please add one, with a bunch of subsections for various "here's why I picked this way over some other way" decisions you made.